### ISSUE TITLE
Onboard information not populated to supplier registration

### ISSUE THREAD

**Issue by gr4per** [OpusCapita/supplier#42]
I have this in user.UserOnboardData:
```
| arne.graeper@opuscapita.com   | {"name":"Arne OHG","vatIdentNo":"DE83412214334","taxIdentNo":null,"dunsNo":null,"commercialRegisterNo":null,"city":"Somewhere","country":"Germany"}             | {"invitationCode":"c8fcb98d-543c-45ff-ad51-1f45e209134e","email":"arne.graeper@opuscapita.com","firstName":"Arne","lastName":"Gräper","campaignId":"testMeAgain","serviceName":"eInvoiceSend"}    | NULL         | NULL            | eInvoiceSend | c8fcb98d-543c-45ff-ad51-1f45e209134e |
```
but form is completely blank when loaded.

**Comment by gr4per** [comment]
I can even see client successfully called the api to get this
![image](https://cloud.githubusercontent.com/assets/22681804/25505031/717dc37a-2ba0-11e7-88b0-b5e76c147b0c.png)

**Comment by lasong** [comment]
Implemented and resolved by #44
